packages/photon/package.json

```diff
@@ -37,11 +42,12 @@
 		"seed-random": "^2.2.0"
 	},
 	"devDependencies": {
-		"@automattic/calypso-typescript-config": "workspace:^"
+		"@automattic/calypso-typescript-config": "workspace:^",
+		"@types/seed-random": "^2.2.1"
```

Comment:
`@types/foo` should be in the same group as `foo` (https://github.com/Automattic/wp-calypso/pull/58711#discussion_r760901539)

Reply:
That's very useful to know, thanks!